src/CMakeLists.txt

```diff
@@ -41,6 +42,7 @@ set(HEADERS helpers.h
 
 add_executable(${TARGET_EXE} main.cc)
 add_library(${TARGET} OBJECT ${SRC})
+add_library(${TARGET_LIB} SHARED ${SRC} api.cc)
```

Comment:
`akite.cc` instead of `api.cc`?

Reply:
i changed the header to `include/aktualizr-lite/api.h` so i think this name makes more sense now.